pandas/tests/frame/test_apply.py

```diff
@@ -116,6 +116,30 @@ def test_apply_with_reduce_empty(self):
         # Ensure that x.append hasn't been called
         assert x == []
 
+    def test_apply_funcs_over_empty(self):
```

Comment:
this would likely need to be parametrize 

we already have lots of tests for this iiirc

Reply:
Looks like there are.  These do seem different in the sense that exactly one axis is non-empty instead of both (so we have different expected output); should the name of the test reflect that?

I made a change to parametrize the tests, although I wasn't sure how to use the same pattern for `nunique`.  Could that go in its own test?